coderd/userauth.go

```diff
@@ -535,9 +536,7 @@ func (api *API) userOAuth2Github(rw http.ResponseWriter, r *http.Request) {
 			}
 		}
 		if len(selectedMemberships) == 0 {
-			httpapi.Write(ctx, rw, http.StatusUnauthorized, codersdk.Response{
-				Message: "You aren't a member of the authorized Github organizations!",
-			})
+			httpmw.RedirectToLogin(rw, r, &url.URL{Path: "/login"}, "You aren't a member of the authorized Github organizations!")
```

Comment:
```suggestion
			httpmw.RedirectToLogin(rw, r, nil, "You aren't a member of the authorized Github organizations!")
```

dashboardURL defaults to `/login` if nil